templates/user/submit/submit.html.twig

```diff
@@ -17,6 +17,14 @@
     <label for="submits_users" class="d-block">Users</label>
 {% endblock %}
 
+{% block _submit_users_widget %}
+    <select name="submit[users][]" id="tom-select" multiple>
```

Comment:
there is a twig variable to get the `name`
And the id is needed? IE this custom ID for the connection between the label and the select

Reply:
This ID is used to transform the select into a tom select, we need a unique ID for this

Reply:
The ID is always unique :)

Reply:
I know, but since it seems a bit complicated to use a class, select them all and transform them all into a tomselect, I decided to instead use an ID and limit the number of tomselects on a page to 1.

You think  we should use a class ?

Reply:
I'm not against a class, but I don't see what's hard here :/ You can have the same behavior with a class. You can add the class to only one element. More over, it's more flexible